py_test.py

```
@@ -0,0 +1,374 @@
+                    'picking_id': picking.id,
+                })
+                line.move_orig_ids=move.ids
+            picking.action_confirm()
+
+            rec.state = 'component_picked'
+
+    @api.model
+    def _generate_mfg_lot_name(self):
+
+
+        current_year = datetime.now().year
```

Ruff (0.11.9)

Local variable `current_year` is assigned to but never used

Remove assignment to unused variable `current_year`

(F841)